xml/System.Buffers/NIndex.xml

```diff
@@ -24,8 +24,14 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Represent a type can be used to index a collection either from the start or the end.</summary>
```

Comment:
Does this version of the docs have the new `Experimental` attribute that was applied to it for .NET 9?

Reply:
Same question applies to `NRange`

Reply:
@tannergooding When was the attribute added? I am asking because currently, dotnet-api-docs is pointing to Preview6.